runtime/doc/options.txt

```diff
@@ -7090,6 +7090,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	UI-dependent. Works best with RGB colors. 'termguicolors'
 
+						*'winborder'* *'winbd'*
+'winborder' 'winbd'	string	(default "")
+			global
+	Configures the border style of floating windows, border values only
+	support string type which defined in |nvim_open_win()|.
```

Comment:
```suggestion
    Defines the default border style of floating windows. See |nvim_open_win()|
    for valid values.
```

Reply:
We should probably move the list of border names here, and link to this from |nvim_open_win()|